core/src/impl/Kokkos_Default_GraphNodeKernel.hpp

```diff
@@ -61,17 +68,19 @@ class GraphNodeKernelImpl
   GraphNodeKernelImpl(std::string const &, ExecutionSpace const &,
                       Functor arg_functor, PolicyDeduced &&arg_policy,
                       ArgsDeduced &&...args)
-      : base_t(std::move(arg_functor), (PolicyDeduced &&)arg_policy,
-               (ArgsDeduced &&)args...),
-        execute_kernel_vtable_base_t() {}
+      : execute_kernel_vtable_base_t(arg_policy.space()),
+        base_t(std::move(arg_functor), (PolicyDeduced &&)arg_policy,
+               (ArgsDeduced &&)args...) {}
 
   // FIXME @graph Forward through the instance once that works in the backends
   template <class PolicyDeduced, class... ArgsDeduced>
   GraphNodeKernelImpl(ExecutionSpace const &ex, Functor arg_functor,
                       PolicyDeduced &&arg_policy, ArgsDeduced &&...args)
       : GraphNodeKernelImpl("", ex, std::move(arg_functor),
                             (PolicyDeduced &&)arg_policy,
-                            (ArgsDeduced &&)args...) {}
+                            (ArgsDeduced &&)args...) {
+    // FIXME This constructor seem unused.
```

Comment:
So is it used?

Reply:
@dalg24 OK to close this thread if:
- I don't delete the constructor because we don't know if someone downstream would be impacted.
- Keep the comment because this is still something worth investigating in the future.

Reply:
How could this constructor be used given that its name contains "Impl"?

Reply:
I don't mind deleting this constructor if you guys think it's OK, but I would refrain doing it here because this PR's purpose is not really to cleanup the mess :wink: